src/main/kotlin/no/nav/dagpenger/regel/api/arena/adapter/v1/GrunnlagOgSatsApi.kt

```diff
@@ -103,14 +103,22 @@ fun mergeGrunnlagOgSatsSubsumsjon(
 }
 
 fun findBeregningsregel(grunnlagResultat: GrunnlagResultat): GrunnlagOgSatsResultat.Beregningsregel {
-    val beregningsregel = grunnlagResultat.beregningsregel
-    return when (beregningsregel) {
-        "ArbeidsinntektSiste12", "FangstOgFiskSiste12" -> GrunnlagOgSatsResultat.Beregningsregel.ORDINAER_OVER_6G_SISTE_2019
-        "ArbeidsinntektSiste36", "FangstOgFiskSiste36" -> GrunnlagOgSatsResultat.Beregningsregel.ORDINAER_OVER_6G_3SISTE_2019
-        "Verneplikt" -> GrunnlagOgSatsResultat.Beregningsregel.VERNEPLIKT
-        "Manuell under 6G" -> GrunnlagOgSatsResultat.Beregningsregel.MANUELL_UNDER_6G
-        "Manuell over 6G" -> GrunnlagOgSatsResultat.Beregningsregel.MANUELL_OVER_6G
-        else -> throw FeilBeregningsregelException("Ukjent beregningsregel: '$beregningsregel'")
+
+    return when {
+        grunnlagResultat.beregningsregel == "Manuell" && grunnlagResultat.harAvkortet -> GrunnlagOgSatsResultat.Beregningsregel.MANUELL_OVER_6G
```

Comment:
siden det er bare to kombinasjoner trenger en ikke sjekke `!grunnlagResultat.harAvkortet` for neste linje, 
`grunnlagResultat.beregningsregel == "Manuell" && grunnlagResultat.harAvkortet` blir alltid evaluert først, så trenger en bare å putte de med ` grunnlagResultat.harAvkortet` først. Da blir det ikke så mye kode